README.md

```diff
@@ -25,8 +25,8 @@ module.exports = {
     new WasmPackPlugin({
       crateDirectory: path.resolve(__dirname, "crate"),
       // check https://rustwasm.github.io/wasm-pack/book/commands/build.html for extraArgs
-      // defautls --typescript --target browser --mode normal
-      extraArgs: "--no-typescript --mode no-install",
+      // defaults `--target browser --mode normal`
```

Comment:
Please, add `--typescript` again as it is a default value for the `wasm-pack build` command.

Reply:
I think `wasm-pack` doesn't support `--typescript` arg. It cause confusion.

https://github.com/rustwasm/wasm-pack/blob/600d02b292083fdd1d7a7ed1bc3e429048feed2c/src/command/build.rs#L78-L81

Reply:
You're totally right! My bad. I saw [this line](https://github.com/rustwasm/wasm-pack/blob/600d02b292083fdd1d7a7ed1bc3e429048feed2c/src/bindgen.rs#L145) and it led me to confussion. 👍